lib/macho/macho_file.rb

```diff
@@ -424,6 +463,16 @@ def low_fileoff
       offset
     end
 
+    # Updates the number of load commands in the raw data.
+    # @param ncmds [Fixnum] the new number of commands
+    # @return [void]
+    # @private
+    def set_ncmds(ncmds)
+      fmt = Utils.specialize_format("L=", endianness)
+      new_ncmds = [ncmds].pack(fmt)
```

Comment:
Oh, I think I misunderstood you. I thought you wanted to change the `ncmds` parameter to `ncmds_raw`, instead of changing `new_ncmds` to `ncmds_raw`. That latter change is 100% okay with me, if that's what you meant.